open_humans/management/commands/vacuum_log_bucket.py

```diff
@@ -0,0 +1,99 @@
+from datetime import datetime
+from urllib.parse import urlparse, parse_qs
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import boto3
+
+from data_import.models import DataFile, AWSDataFileAccessLog, NewDataFileAccessLog
+
+
+class Command(BaseCommand):
+    """
+    A management command vaccuming up file access logs and putting them in the database.
+    """
+
+    help = "Populates database with file access logs"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Retreiving file access logs")
+        s3 = boto3.resource("s3")
+        bucket = s3.Bucket(settings.LOG_BUCKET)
+        for item in bucket.objects.all():
+            log_objects = item.get()["Body"].read().decode("utf-8").split("\n")
+            for log_entry in log_objects:
+                if not log_entry:
+                    continue
+                log = []
+                start_chars = ['"', "["]
```

Comment:
It seems unlikely that you should need to write a new "quote-protected split" yourself. I googled `quote-protected split python` and found this?

https://stackoverflow.com/questions/79968/split-a-string-by-spaces-preserving-quoted-substrings-in-python

Reply:
Ahhh, your Google-foo is better than mine!